### PR TITLE
RHIDP-2067 [QE] Automate Scenario - Support external postgres databases

### PR DESCRIPTION
**Pull request by gustavolira**
## Description
Manual test to verify external PostgreSQL installation.
https://issues.redhat.com/browse/RHIDP-2067

Created a manual-tests folder which will contain all new manual tests from now on. This will help identify which manual tests need to be run.

## Which issue(s) does this PR fix

- Fixes #?

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: .ibm/manual-tests/postgres/install.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+set -x
+
+HELM_IMAGE_NAME=backstage
+HELM_REPO_NAME=rhdh-chart
+HELM_REPO_URL="https://redhat-developer.github.io/rhdh-chart"
```
Comment (rnapoles-rh) [inline .ibm/manual-tests/postgres/install.sh:8]: Will this use the production helm chart?

File: .ibm/manual-tests/postgres/postgres-crt-secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-crt
+type: Opaque
+stringData:
+  postgres-crt.pem: |-
+    -----BEGIN CERTIFICATE-----
```
Comment (josephca) [inline .ibm/manual-tests/postgres/postgres-crt-secrets.yaml:8]: @gustavolira can we unexpose the credentials?

**Review by josephca — DISMISSED** [review]

File: .ibm/manual-tests/postgres/postgres-crt-secrets.yaml (hunk 2 above)
Comment (gustavolira) [reply .ibm/manual-tests/postgres/postgres-crt-secrets.yaml:8]: > @gustavolira can we unexpose the credentials?

@josephca  the value of the certificate is a dummy value, I only downloaded it from the site and get the value, we don't have any sensitive information in this certificate

File: .ibm/manual-tests/postgres/install.sh (hunk 1 above)
Comment (gustavolira) [reply .ibm/manual-tests/postgres/install.sh:8]: @rnapoles-rh we can change the helm chart for whatever we want change only changing the variables HELM_REPO_URL, HELM_REPO_NAME,  HELM_IMAGE_NAME

Comment (rnapoles-rh) [reply .ibm/manual-tests/postgres/install.sh:8]: Got it, thx

**Pull request merged**